api/database/factories/PoolFactory.php

```diff
@@ -164,4 +165,17 @@ public function candidatesAvailableInSearch()
             ];
         });
     }
+
+    /** Add assessment steps to the pool
+     *
+     * @return \Illuminate\Database\Eloquent\Factories\Factory
+     */
+    public function withAssessments()
+    {
+        return $this->afterCreating(function (Pool $pool) {
+            AssessmentStep::factory()
```

Comment:
I think this withAssessments method should manually create the Application Screening step, and the Screening Question step, if $pool has any screening questions.

Reply:
I don't know, I think it should be strictly controlled by the methods that will be responsible for it in #8034
Defining it in factory too would create a redundancy wouldn't it? Running `Pool::factory()->withAssessments()` after that is run would have the restricted assessments steps called/made twice.

Reply:
Excellent point! Never mind then.